python/zensical/config.py

```diff
@@ -456,6 +456,15 @@ def _apply_defaults(config: dict, path: str) -> dict:
 
     # Convert plugins configuration
     config["plugins"] = _convert_plugins(config.get("plugins", []), config)
+
+    # mkdocstrings configuration
+    if "mkdocstrings" in config["plugins"]:
+        mkdocstrings_config = config["plugins"]["mkdocstrings"]["config"]
+        if mkdocstrings_config.pop("enabled", True):
+            mkdocstrings_config["markdown_extensions"] = [{ext: mdx_configs.get(ext, {})} for ext in markdown_extensions]
```

Comment:
I'm rebuilding a dict from splitted names/configs here. I could modify the mkdocstrings extension to accept names and configs separately, to avoid rebuilding here, if we decide we want the perf increase (which I assume is very minimal).